WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -140,7 +140,7 @@ - (void)viewDidLoad
 
     [WPStyleGuide resetReadableMarginsForTableView:self.tableView];
     [WPStyleGuide configureColorsForView:self.view andTableView:self.tableView];
-    [self.tableView registerClass:[WPTableViewCell class] forCellReuseIdentifier:BlogDetailsCellIdentifier];
+    [self.tableView registerClass:[WPTableViewCellValue1 class] forCellReuseIdentifier:BlogDetailsCellIdentifier];
```

Comment:
The issue with the comment count comes from having the same cell identifier. You need to register both `WPTableViewCell` (for `BlogDetailsCellIdentifier`), and `WPTableViewCellValue1` for a new identifier used only by the plans cell. [I normally use the class name](https://github.com/wordpress-mobile/WordPress-iOS/blob/371e215d487816b469fc736f9b17d49ad2c68599/WordPress/Classes/Utility/ImmuTable.swift#L195)  as the cell identifier to make sure I get what I want.